org.eclipse.scanning.api/src/org/eclipse/scanning/api/points/models/AbstractMapModel.java

```diff
@@ -70,8 +70,10 @@ public boolean isContinuous() {
 	}
 
 	@Override
-	public void setContinuous(boolean continuous) {
-		this.continuous = continuous;
+	public void setContinuous(boolean newValue) {
+		boolean oldValue = this.continuous;
+		this.continuous = newValue;
+		this.pcs.firePropertyChange("continuous", oldValue, newValue);
 	}
 
 	@UiHidden
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "UiHidden"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)